LibCore/CProcessHelper.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <AK/AKString.h>
+#include <AK/HashMap.h>
+#include <AK/RetainPtr.h>
+
+class CProcessInfo : public Retainable<CProcessInfo> {
```

Comment:
Info is a bit too generic, let's call this CProcessStatistics.
Also, it doesn't need to be a retainable class at all, since it's not expected to persist with multiple owners.
I would suggest making it a simple value struct:

class CProcessStatistics {
    ....
};